.github/workflows/ci.yml

```
@@ -76,6 +76,9 @@ jobs:
         # Install pybind11
         ./vcpkg.exe install --overlay-ports=C:/robotology/robotology-vcpkg-binary-ports pybind11:x64-windows
 
+        # Install spdlog
+        ./vcpkg.exe install --overlay-ports=C:/robotology/robotology-vcpkg-binary-ports spdlog
```

```suggestion
        ./vcpkg.exe install --overlay-ports=C:/robotology/robotology-vcpkg-binary-ports spdlog:x64-windows
```

Otherwise the 32 bit version is installed.